plugins/imfile/imfile.c

```diff
@@ -2846,6 +2879,7 @@ resetConfigVariables(uchar __attribute__((unused)) *pp, void __attribute__((unus
 	cs.readMode = 0;
 	cs.maxLinesAtOnce = 10240;
 	cs.trimLineOverBytes = 0;
+	cs.ignoreOlderThan = 0;
```

Comment:
please remove support for the old config system - we do NOT provide new features via the old system.